src/accordion/accordion.ts

```diff
@@ -209,6 +209,13 @@ export class NgbAccordion implements AfterContentChecked {
    */
   @Input() type: string;
 
+  /**
+   * The panel's role attribute value. According to
+   * [WAI-ARIA practices](https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties)
+   * could be a '[region](https://www.w3.org/TR/wai-aria-1.1/#region)'
+   */
+  @Input() ariaPanelRole: 'region' | null;
```

Comment:
I am still not convinced, as said previously, that we should introduce new Input for that.
Is there an actual need to change that? Should it always be a `panel` ? If yes, there's no need for the input. Let's change the previous `role="tab panel"` to the new one `role="region"`.

Reply:
Hi @benouat !
I totally understand your concerns and happy to discuss since I don't have 100% confidence here either.

the "region" role determines if the content should appear in the Landmarks navigation menu. It is one of menus opened by VO+U.

here is the extraction from [WAI-ARIA practices for accordion widget](https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties):

> Optionally, each element that serves as a container for panel content has role region and aria-labelledby with a value that refers to the button that controls display of the panel.
> - Avoid using the region role in circumstances that create landmark region proliferation, e.g., in an accordion that contains more than approximately 6 panels that can be expanded at the same time.
> - Role region is especially helpful to the perception of structure by screen reader users when panels contain heading elements or a nested accordion.

so, the need for this role depends on specific usage of the widget and only developer could determine when add role region and when not. 

we could consider avoid this to be part of accordion component and be part of the content template. `Optionally ... a container for panel content has role region and aria-labelledby ...`. It doesn't feel aligned with practices to me if the `aria-labelledby` would be in one HTML element and `role="region"` in the descendant. When it is one element, the region landmark  name would be calculated using `aria-labelledby` attribute.

Reply:
I would be thankful for the feedback (: